### ISSUE TITLE
Transform,replace xarg_ acts only on symbols

### ISSUE THREAD

**Issue by apik** [vermaseren/form#54]
I want to apply simple transformation to all function arguments, but only symbolic arguments are affected
```
CF f;
S a;

L xx = f(a,1);

Transform,f,replace(1,last)=(xarg_,2*xarg_);

```

Produce 
```
f(2*a,1)
```

instead of 
```
f(2*a,2)
```

**Comment by vermaseren** [comment]
Sorry for the late reply.
I just fixed it.

    CF f;
    S a;
    L xx = f(a,a^2,1,2);
    Transform,f,replace(1,last)=(xarg_,2*xarg_,1,3);
    Print;
    .end

   xx =
      f(2*a,2*a^2,3,4);

Thanks for the report.

Jos

**Comment by tueda** [comment]
Fixed in 07edf537394f66d12ad234f4e058f04fb2bc9d08.
